teachertool/src/state/actions.ts

```diff
@@ -84,6 +84,11 @@ type SetAutorun = ActionBase & {
     autorun: boolean;
 };
 
+type SetEvalResultsBatch = ActionBase & {
```

Comment:
`SET_EVAL_RESULTS` vs `SET_EVAL_RESULT` is a very subtle difference. Recommend calling this one `SET_EVAL_RESULTS_BATCH`.